example-playlist.yaml

```diff
@@ -6,7 +6,7 @@ obs_studio:
   preroll_time: 5             # The number of seconds before an obs_video track that the video is loaded and transitioned to
 magicq:
   host: 127.0.0.1
-  port: 6553  # default Chamsys MagicQ port
+  port: 53000  # default Chamsys MagicQ OSC port
```

Comment:
Does this mean that previous playlists are no longer compatible with the implementation?
If so we should probably have a new top level key and backend type (perhaps `magicq-osc` in place of `magicq`; `MagicqOSCController` in place of `MagicqController`), even if we do remove the old one.

My concern is not so much that the old playlists be kept working, but rather than it's clear that they won't work and are rejected cleanly (rather than emitting an unclear error about the commands not working, or, worse, the commands silently failing).

Reply:
I'd also be happy with a version field here.